inst/Upgrades.qml

```diff
@@ -0,0 +1,171 @@
+
+		ChangeRename	{	from: 	"groupNames";						to:		"manualColorGroups"								}
+
+		ChangeJS
+		{
+			name: "manualColorGroups"
+			jsFunction: function(options)
+			{
+
+				return options["manualColorGroups"].map(item=>{
+					item["name"] = item["group"];
+					item["color"] = item["groupColor"];
+					return item;
+				})
+			}
+		}
+
+		ChangeRename	{	from:	"variablesForColor";				to:		"colorGroupVariables"							}
+		ChangeJS
+		{
+			name: "colorGroupVariables"
+			jsFunction: function(options)
+			{
+
+				return options["colorGroupVariables"].map(item=>{
+					item["group"] = item["groupAssigned"];
+					return item;
+				})
+
+			}
+		}
+
+
+		ChangeRename	{	from:	"manualColors";						to:		"manualColor"									}
+
+		ChangeRename	{	from:	"showDetails";						to:		"details"										}
+
+		ChangeRename	{	from:	"edgeLabelCex";						to:		"edgeLabelSize"									}
+		ChangeRename	{	from:	"edgeColors";						to:		"edgePalette"									}
+
+		ChangeJS
+		{
+			name: "edgePalette"
+			jsFunction: function(options)
+			{
+				switch (options["edgePalette"])
+				{
+					case "Hollywood":			return "hollywood";
+					case "Borkulo":				return "borkulo";
+					case "TeamFortress":		return "teamFortress";
+					case "Reddit":				return "reddit";
+					case "Fried":				return "fried";
+					// the other values did not change
+					default:					return options["edgePalette"];
+
+				}
+			}
+		}
+
+		ChangeRename	{	from:	"scaleLabels";						to:		"labelScale"									}
+		ChangeRename	{	from:	"abbreviateLabels";					to:		"labelAbbreviation"								}
+		ChangeRename	{	from:	"abbreviateNoChars";				to:		"labelAbbreviationLength"						}
+
+		ChangeRename	{	from:	"showVariableNames";				to:		"variableNamesShown"							}
+		ChangeJS
+		{
+			name: "variableNamesShown"
+			jsFunction: function(options)
+			{
+				switch (options["variableNamesShown"])
+				{
+					case "In nodes":			return "inNodes";
+					case "In legend":			return "inLegend";
+				}
+			}
+		}
+
+		ChangeRename	{	from:	"showMgmVariableType";				to:		"mgmVariableTypeShown"							}
+		ChangeJS
+		{
+			name: "mgmVariableTypeShown"
+			jsFunction: function(options)
+			{
+				switch (options["mgmVariableTypeShown"])
+				{
+					case "mgmNoShow":			return "hide";
+					case "mgmNodeColor":		return "nodeColor";
+					case "mgmNodeShape":		return "nodeShape";
+				}
+			}
+		}
+
+		ChangeRename	{	from:	"showLegend";						to:		"legend"										}
+		ChangeJS
+		{
+			name: "legend"
+			jsFunction: function(options)
+			{
+				switch (options["legend"])
+				{
+					case "No legend":			return "hide";
+					case "All plots":			return "allPlots";
+					case "In plot number: ":	return "specificPlot";
+				}
+			}
+		}
+
+		ChangeRename	{	from: 	"legendNumber";						to:		"legendSpecificPlotNumber"						}
+		ChangeRename	{	from: 	"keepLayoutTheSame";				to:		"layoutNotUpdated"								}
+		ChangeRename	{	from: 	"repulsion";						to:		"layoutSpringRepulsion"							}
```

Comment:
you forgot to change this option name :)

Reply:
oops sorry about that :sweat_smile: